app/src/main/res/values/strings.xml

```diff
@@ -92,6 +92,9 @@
     </string-array>
 
     <!-- Content Description -->
+    <string name="connection">Connected</string>
+    <string name="fastscroller_track">FastScroller track</string>
```

Comment:
I don't see why those strings are here

Reply:
I think those result from a merge from OFF upstream to his development branch